SocialNetwork/SocialNetwork.Core/Models/Post.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace SocialNetwork.Core.Models
+{
+    public class Post
+    {
+        public int Id { get; set; }
+        public string? Description { get; set; }
+        public object? Media { get; set; }
```

Comment:
object isn't the best data type for images. Better to use string and store a link to the resource that contains media content